### ISSUE TITLE
About the Local Deployment of Cloned tfjs

### ISSUE THREAD

**Issue by Zengyf-CVer** [tensorflow/tfjs#5365]
@lina128 

Hello, first of all thank you for fixing my problem(https://github.com/tensorflow/tfjs/issues/5291 & https://github.com/tensorflow/tfjs/pull/5359).
I am excited to try the code, but I have encountered some problems, please help me, thank you.
I cloned the latest version of the tfjs project:

```shell
git clone https://github.com/tensorflow/tfjs.git
```

I modified the `package.json` file in my object detection code:(here I used local configuration："file:path/to/tfjs...")
```json
    "@tensorflow/tfjs": "file:/media/zyf/office/tfjs/tfjs",
    "@tensorflow/tfjs-core": "file:/media/zyf/office/tfjs/tfjs-core",
    "@tensorflow/tfjs-backend-webgl": "file:/media/zyf/office/tfjs/tfjs-backend-webgl",
    "@tensorflow/tfjs-backend-cpu": "file:/media/zyf/office/tfjs/tfjs-backend-cpu",
    "@tensorflow/tfjs-converter": "file:/media/zyf/office/tfjs/tfjs-converter",
    "@tensorflow/tfjs-data": "file:/media/zyf/office/tfjs/tfjs-data",
    "@tensorflow/tfjs-layers": "file:/media/zyf/office/tfjs/tfjs-layers",
```

Then I used the following instructions to install and run:
```shell
yarn
yarn start
```

But the following error appeared in the chrome console:
![ksnip_20210722-220418](https://user-images.githubusercontent.com/41098760/126654962-ac3d01e9-cb07-4da9-a7f7-ec836edf2043.png)

But there are indeed related files in the `node_modules` folder:
![ksnip_20210722-222139](https://user-images.githubusercontent.com/41098760/126655353-e5aa2dce-dd4e-445f-aeeb-abd19ff40da3.png)
So I don't know where there is an error in my execution steps, please help me, thank you.

**Comment by rthadur** [comment]
@Zengyf-CVer may i know why you are modifying package.json ? tfjs is a union package which has most of tfjs-layers , tfjs-converter etc ?so please try to keep one package reference.

**Comment by Zengyf-CVer** [comment]
@rthadur 

The `package.json` file I modified is under my personal project, because if I use the default tfjs dependency, such as: `"@tensorflow/tfjs": "latest"`, I can only download the released version of `tfjs- 3.8.0`, and it was released seven days ago.

I want to use the real-time, the latest, but not the release version, so I need to clone it, and then modify it in the `package.json` in my personal project to link the dependency of tfjs to the real-time tfjs I cloned Project.

This is `package.json` under my personal project:
![ksnip_20210723-102453](https://user-images.githubusercontent.com/41098760/126730871-bdbf005d-521b-44c6-837a-5859038dd339.png)

**Comment by rthadur** [comment]
got it ,this is not an issue with tfjs , i see a problem while referencing it in package.json, please check a related issue here https://stackoverflow.com/questions/33816241/add-local-project-dependency-for-npm-install. Thank you
